src/schema.ts

```diff
@@ -43,15 +46,30 @@ export class Schema {
         }
       } else if (primitiveType === PrimitiveType.DATE) {
         type = Type.TEMPORAL;
+        // need to get correct min/max of date data
```

Comment:
> need to get correct min/max of date data

Why?  Is it because datalib doesn't infer this correctly?  I thought you say it's not a problem anymore.